openapi/p4pa-workflow-hub.openapi.yaml

```
@@ -6,6 +6,36 @@ info:
 servers:
   - url: "http://localhost:8080/workflowhub"
 paths:
+  /PaymentIngestionWF/{ingestionFileId}:
+    post:
+      tags:
+        - WorkflowHub
```

let's actually partitionate the controller on the generic IngestionFlow
```suggestion
        - IngestionFlow
```

---

Done